### ISSUE TITLE
Ambiguous key sequences (precedence problem?)

### ISSUE THREAD

**Issue by markmr** [eteran/nedit-ng#107]
Hello,
I have created a small macro to do some regular task (remove trailing blanks at line ends) and assigned it to "Alt+s". (This example is for demonstration purposes. Of course you may ask, why not take another key sequence...)

![ScreenShot4a](https://user-images.githubusercontent.com/6682033/58495903-f7811000-8178-11e9-9f6e-44339d4412c2.png)

1) Pressing Alt+s does not invoke my macro but highlights the "Search" menu. Seems this is hardwired somewhere.

![ScreenShot5](https://user-images.githubusercontent.com/6682033/58495961-1aabbf80-8179-11e9-872d-0fba2111d7eb.png)

2) Pressing Alt+s again (Esc between) results in a error message as shown in picture 3. 

![ScreenShot6](https://user-images.githubusercontent.com/6682033/58495986-28614500-8179-11e9-8677-66838022b882.png)
(For me this message sounds more as a desktop configuration error, not as a application error. But this is not the problem here.)

3) Pressing Alt+s again (Esc between) hightlights the "Search" menu as in 1. And so on and on ....

Markus

**Comment by eteran** [comment]
Yes, at the moment the shortcuts are indeed hardcoded in the UI. (I may already have an issue for it, not sure).

Basically I am a little unsure how to best handle things like shift shortcuts and user configurable shortcuts. Do I let users make shift versions have nothing to do with non-shift versions? If so, how do I decided when to display the "[shift]" in the menu? Etc...

But rest assured, I am thinking on this!
